code/modules/mob/living/carbon/human/update_icons.dm

```diff
@@ -377,6 +383,9 @@ var/global/list/damage_icon_parts = list()
 	else
 		//BEGIN CACHED ICON GENERATION.
 		var/obj/item/organ/external/chest = get_organ(BP_CHEST)
+		if(!chest)
+			return // This should not be happening, but occasionally there is an ordering issue or a late call to update icon.
```

Comment:
It was happening during testing and I'm really not clear on why. I assume because some part of droplimb is forcing a body update - it's possible now that it's queued it won't be an issue. I might add a PRINT_STACK_TRACE before returning.

Reply:
Turns out we can completely remove the need for a root organ.